Sources/NinetyNineSwiftProblems/Graph.swift

```diff
@@ -1,18 +1,18 @@
 import Foundation
 
 class Graph<T, U> {
-    var direction: GraphDirection { return .Indirected }
+    var direction: GraphDirection { return .indirected }
     var nodes: List<T>?
     var edges: List<(T, T, U)>?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/eneko/Ninety-Nine-Swift-Solutions/blob/10b133b7b99f71c99d3d5c8d75d48801cb5e6588/Sources/NinetyNineSwiftProblems/Graph.swift#L6"></span>Tuples should have at most 2 members.</td>
    </tr>
  </tbody>
</table>